apps/k8s-io/configmap-nginx.yaml

```diff
@@ -152,13 +152,15 @@ data:
         server_name apt.kubernetes.io apt.k8s.io;
         listen 80;
 
+        rewrite ^/$         https://kubernetes.io/blog/2023/08/31/legacy-package-repository-deprecation/ redirect;
         rewrite ^/(.*)?$    https://packages.cloud.google.com/apt/$1 redirect;
       }
 
       server {
         server_name yum.kubernetes.io yum.k8s.io;
         listen 80;
 
+        rewrite ^/$         https://kubernetes.io/blog/2023/08/31/legacy-package-repository-deprecation/ redirect;
```

Comment:
just confirming: this appears to serve 302? cc @sftim

Reply:
Yes, we have the same redirect in place for `pkgs.k8s.io` and it is 302:

```
< HTTP/2 302
< server: nginx
```

We can replace `redirect` with `permanent` to serve 301 redirects. I don't have strong opinions, it's very easy to change this. 🙂

Reply:
I'd use a 302 redirect, because we're not committing that that blog article is the definitive new location for all time (eg we might make a short URL and decide to redirect via that, or we might even make a microsite about legacy things)